src/twisted/names/test/test_hosts.py

```diff
@@ -203,13 +203,13 @@ def test_lookupAddress(self) -> None:
         records from the hosts file.
         """
         d = self.resolver.lookupAddress(b"multiple")
-        answers, authority, additional = self.successResultOf(d)
```

Comment:
On second thought, … how is this not resulting in a Mypy error, given that this test *is* annotated? It looks like maybe the problem is that `twisted.names.common.ResolverBase` itself doesn't have annotations, so the type-check in its `@implements` declaration is incorrectly passing because `Any` is propagating all over the place for argument & return types…